experimental/CHANGELOG.md

```diff
@@ -10,6 +10,8 @@ All notable changes to experimental packages in this project will be documented
 
 ### :bug: (Bug Fix)
 
+* fix: remove HTTP/HTTPS prefix from span name in instrumentation-xml-http-request
```

Comment:
As with the other change, this should probably be listed as a breaking, albeit driven by a spec / semantic convention change.

Reply:
@MSNev I added it in breaking change too